src/Constants.ts

```diff
@@ -13,13 +13,13 @@ export const selectJettyDirectory: string = 'Select Jetty Directory';
 
 export const deleteConfirm: string = 'This Jetty Server is running, are you sure you want to delete it?';
 
-export const serverRunning: string = 'This Jetty Serverrver is running.';
```

Comment:
Naming convention suggestion, for constant value, it should be all capital letter with underscore